CarRentals/CarRentals/Controllers/CarCRUD.cs

```diff
@@ -87,7 +81,6 @@ public void Delete(int id)
             SaveChanges();
         }
 
-
         public void ReadJson()
```

Comment:
any reason to make this method public?
also, I'm not sure about having this try catch here, what do you think the desired behavior should be if there's an error while opening/reading the JSON file? and how's that compared to the current behavior?

Reply:
I corrected the acces modifier to private in this method and in SaveChanges()